drivers/bmx280/bmx280.c

```diff
@@ -23,93 +25,320 @@
 #include <math.h>
 
 #include "log.h"
+#include "assert.h"
 #include "bmx280.h"
 #include "bmx280_internals.h"
-#include "bmx280_params.h"
-#include "periph/i2c.h"
 #include "xtimer.h"
 
 #define ENABLE_DEBUG        (0)
 #include "debug.h"
 
-static int read_calibration_data(bmx280_t* dev);
-static int do_measurement(const bmx280_t* dev);
-static uint8_t get_ctrl_meas(const bmx280_t* dev);
-static uint8_t get_status(const bmx280_t* dev);
-static uint8_t read_u8_reg(const bmx280_t* dev, uint8_t reg);
-static void write_u8_reg(const bmx280_t* dev, uint8_t reg, uint8_t b);
-static uint16_t get_uint16_le(const uint8_t *buffer, size_t offset);
-static int16_t get_int16_le(const uint8_t *buffer, size_t offset);
-
-#if ENABLE_DEBUG
-static void dump_buffer(const char *txt, uint8_t *buffer, size_t size);
-#define DUMP_BUFFER(txt, buffer, size)  dump_buffer(txt, buffer, size)
+#ifdef BMX280_USE_SPI
+#define BUS                 (dev->params.spi)
+#define CS                  (dev->params.cs)
+#define CLK                 SPI_CLK_5MHZ
```

Comment:
I think this should go to `bmx280_params.h`

Reply:
Makes sense -> fixed.